adapters/triplelift/usersync_test.go

```diff
@@ -8,11 +8,11 @@ import (
 )
 
 func TestTripleliftSyncer(t *testing.T) {
-	temp := template.Must(template.New("sync-template").Parse("//eb2.3lift.com/sync?gdpr={{.GDPR}}&cmp_cs={{.GDPRConsent}}"))
+	temp := template.Must(template.New("sync-template").Parse("//eb2.3lift.com/getuid?gdpr={{.GDPR}}&cmp_cs={{.GDPRConsent}}&redir=https%3A%2F%2Feb2.3lift.com%2Fsync%3Fbidder%triplelift%26gdpr%3D{{.GDPR}}%26gdpr_consent%3D{{.GDPRConsent}}%26uid%3D%24UID"))
```

Comment:
It looks like the redirect URL has an incomplete escape character sequence. In this test, it is:
`bidder%triplelift`
.. and in the config it is ...
`bidder%3Dtriplelift%`

Reply:
Unfilled extension should be `...%2Fsetuid%3Fbidder%3Dtriplelift%26gdpr%3D{{.GDPR}}%26gdpr_consent%3D{{.GDPRConsent}}%26uid%3D%24UID` in both test and config now.

Reply:
Looks good. Thank you. I'll ask @mansinahar to provide the second approval.